force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js

```diff
@@ -0,0 +1,318 @@
+import { LightningElement, api, wire, track } from 'lwc';
+import commonAmount from '@salesforce/label/c.commonAmount';
+import RDCL_Frequency from '@salesforce/label/c.RDCL_Frequency';
+import lblStatus from '@salesforce/label/c.lblStatus';
+import firstDonation from '@salesforce/label/c.firstDonation'
+import nextDonation from '@salesforce/label/c.nextDonation';
+import mostRecentDonation from '@salesforce/label/c.mostRecentDonation';
+import lastModified from '@salesforce/label/c.lastModified';
+import RD2_ViewMoreDetails from '@salesforce/label/c.RD2_ViewMoreDetails';
+import RD2_ViewLessDetails from '@salesforce/label/c.RD2_ViewLessDetails';
+import updatePaymentMethod from '@salesforce/label/c.updatePaymentMethod';
+import changeAmountOrFrequency from '@salesforce/label/c.changeAmountOrFrequency';
+import stopRecurringDonation from '@salesforce/label/c.stopRecurringDonation';
+import getData from '@salesforce/apex/RD2_ETableController.getData';
```

Comment:
Can we rename this controller method to something more descriptive? `getTableView` or `retrieveTableView` maybe?

Reply:
renamed the method to `retrieveTableView`!